### PR TITLE
No transform

### PR DESCRIPTION
**Pull request by elray1**
This PR adds a variation on the gbq_qr model that is fit without an initial fourth root data transformation.

Tests passing on my laptop:

```
elray@SPH-C02FV0TPMD6 flusion % docker run -it \         
    -w /flusion/code/gbq \
    -v ./retrospective-hub:/flusion/retrospective-hub \
    -v ./submissions-hub:/flusion/submissions-hub \
    flusion pytest
================================================= test session starts =================================================
platform linux -- Python 3.11.0, pytest-7.4.0, pluggy-1.0.0
rootdir: /flusion/code/gbq
collected 2 items                                                                                                     

tests/test_drop_level_feats.py .                                                                                [ 50%]
tests/test_gbq.py .                                                                                             [100%]

============================================ 2 passed in 87.96s (0:01:27) =============================================
```

### CODE REVIEW

File: code/data-pipeline/src/data_pipeline/loader.py [hunk 1]
```diff
@@ -471,23 +476,27 @@ def load_data(self, sources=None, flusurvnet_kwargs=None, hhs_kwargs=None, iline
     # - divide by location- and source- specific 95th percentile
     # - center relative to location- and source- specific mean
     #   (note non-standard order of center/scale)
-    df['inc_4rt'] = (df['inc'] + 0.01)**0.25
-    df['inc_4rt_scale_factor'] = df \
+    if power_transform is None:
+        df['inc_trans'] = df['inc']
```
Comment (elray1) [inline code/data-pipeline/src/data_pipeline/loader.py:480]:
```suggestion
        df['inc_trans'] = df['inc'] + 0.01
```

File: code/gbq/retrospective-experiments/gbq_qr_no_transform.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# This script generates predictions for the gbq_qr_no_transform model for all reference dates.
+# Retrospective model fits are generated using the data that would have been available in
+# real time.
+#
+# To maintain transparency about which model outputs were and were not generated in
+# real time, these model outputs are stored in flusion/retrospective-hub.
+
+# This script should be run with code/gbq as the working directory:
+# python retrospective-experiments/gbq_qr_no_transform.py
```
Comment (bsweger) [inline code/gbq/retrospective-experiments/gbq_qr_no_transform.py:9]: I'm not falling for that again!

File: code/gbq/retrospective-experiments/gbq_qr_no_transform.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        for i in range(29)]
+
+output_root = '../../retrospective-hub/model-output'
+
+commands = [f'python gbq.py --ref_date {ref_date} --output_root {output_root} --model_name gbq_qr_no_transform' \
+                for ref_date in missing_ref_dates]
+
+with Pool(processes=2) as pool:
+    freeze_support()
+    pool.map(run_command, commands)
```
Comment (bsweger) [inline code/gbq/retrospective-experiments/gbq_qr_no_transform.py:32]:
```suggestion
def main():
    with Pool(processes=2) as pool:
        freeze_support()
        pool.map(run_command, commands)

if __name__ == '__main__':
    freeze_support()
    main()
```

Comment (bsweger) [reply code/gbq/retrospective-experiments/gbq_qr_no_transform.py:32]: The construct in the above suggestion is what got things working on my machine. From the [docs](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.freeze_support):

> One needs to call this function straight after the if __name__ == '__main__' line of the main module. 

None of this explains why I'd be seeing this if you haven't "frozen" anything, but the suggested change got the script to run.

**Comment by elray1** [comment]
I committed the suggested updates and confirmed that the script still generates the same outputs on my computer.

**Pull request merged**